service/managedinstance/providers/aws/aws.go

```diff
@@ -305,6 +305,23 @@ type DeleteManagedInstanceInput struct {
 
 type DeleteManagedInstanceOutput struct{}
 
+type StatusManagedInstanceInput struct {
+	ManagedInstanceID *string `json:"managedInstanceId,omitempty"`
+}
+
+type StatusManagedInstanceOutput struct {
+	ID           *string    `json:"id,omitempty"`
+	ImageId      *string    `json:"imageId,omitempty"`
+	InstanceId   *string    `json:"instanceId,omitempty"`
+	InstanceType *string    `json:"instanceType,omitempty"`
+	Name         *string    `json:"name,omitempty"`
+	PrivateIp    *string    `json:"privateIp,omitempty"`
+	PublicIp     *string    `json:"publicIp,omitempty"`
```

Comment:
```suggestion
	PublicIP     *string    `json:"publicIp,omitempty"`
```

[Ref](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).